draft-ietf-jsonpath-base.md

```diff
@@ -622,7 +621,7 @@ of input node.
 ### Syntax
 {: unnumbered}
 
-Every JSONPath query MUST begin with the root identifier `$`.
+Every JSONPath query (except those inside filter expressions, see {{filter-selector}}) MUST begin with the root identifier `$`.
```

Comment:
Do we call the things inside the filter expressions (filter-path/singular-path) "JSONPath query"?

Reply:
Yes, I think we do. Section 1.4 says:
> A JSONPath expression consists of an identifier followed by a series of zero or more segments each of which contains one or more selectors.

and "query" is just a short name for "JSONPath expression".

Reply:
Hmm, wouldn't it be useful to have a name for the thing that can be a JSONPath query outside a filter expression?

Reply:
Hmm ... there is only a single JSONPath expression outside of filter expressions.

We also should obey our own terminology:

Query:   Short name for a JSONPath expression.